### ISSUE TITLE
Class 'ZipArchive' not found when trying to download archive in the backend

### ISSUE THREAD

**Issue by mhuijser** [austintoddj/canvas#245]
Hi,

Just for the record. I got the following error when trying to download the archive from the backend.

```
Whoops, looks like something went wrong.

1/1
FatalThrowableError in ToolsController.php line 79:
Class 'ZipArchive' not found
in ToolsController.php line 79
```

It apppears that i needed to install the php zip module on my webserver. (eg. by invoking `apt-get install php7.0-zip` (ubuntu)) Probably a prerequisite to mention in your installation manual.

Thanks for the nice application!

Kind regards,

Mark

**Comment by austintoddj** [comment]
Thanks @mhuijser! Good catch on that. I'll make this change as soon as soon as #244 is fixed. Glad you like the application!

**Comment by mhuijser** [comment]
Hard one. I worked around the #244 prob by hardcoding the Laravel-version (5.3.20) into  in the composer.json before invoking composer update/install. I know, a bit ugly, but working for now.

**Comment by austintoddj** [comment]
Yeah for sure. Kinda just waiting on Proengsoft to get their stuff fixed. Emailed them and they have an open issue, so it's in their court now. Hope anyone else experiencing this breaking change with Canvas finds the temporary solution for now.

**Comment by austintoddj** [comment]
Fixed with dec1749cf692fe1158ac3f84fd06d6c92d77b5ef
